sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/util/TelemetryOptions.java

```diff
@@ -0,0 +1,62 @@
+package com.azure.digitaltwins.core.util;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.UUID;
+
+/**
+ * The additional information to be used when processing a telemetry request.
+ */
+@Fluent
```

Comment:
@abhipsaMisra mentioned earlier that request options objects like these should not be fluent and should not be marked as fluent.

Reply:
no, request options are considered model classes (input/output to API surface) so they should be fluent ... I confirmed it with Srikanta.

Reply:
Yeah, Azad synced up with Srikanta, and he explained that from client libraries perspective, everything that is q request/ response in our public API is a model. It doesn't have anything to do with the models that we actually send over the wire.